rllib/algorithms/callbacks.py

```diff
@@ -283,46 +272,40 @@ def on_episode_created(
-        worker: Optional["EnvRunner"] = None,
         env_runner: Optional["EnvRunner"] = None,
-        # TODO (sven): Deprecate this arg new API stack (in favor of `env`).
-        base_env: Optional[BaseEnv] = None,
+        metrics_logger: Optional[MetricsLogger] = None,
         env: Optional[gym.Env] = None,
-        # TODO (sven): Deprecate this arg new API stack (in favor of `rl_module`).
-        policies: Optional[Dict[PolicyID, Policy]] = None,
-        rl_module: Optional[RLModule] = None,
         env_index: int,
+        rl_module: Optional[RLModule] = None,
+        # TODO (sven): Deprecate these args.
+        worker: Optional["EnvRunner"] = None,
+        base_env: Optional[BaseEnv] = None,
+        policies: Optional[Dict[PolicyID, Policy]] = None,
         **kwargs,
     ) -> None:
-        """Callback run right after an Episode has started.
+        """Callback run right after an Episode has been started.
 
-        This method gets called after a new Episode(V2) (old stack) or
-        SingleAgentEpisode/MultiAgentEpisode instance has been reset via the
-        `env.reset()` API by RLlib.
+        This method gets called after a SingleAgentEpisode or MultiAgentEpisode instance
+        has been reset with a call to `env.reset()` by the EnvRunner.
 
-        1) Episode(V2)/Single-/MultiAgentEpisode created: on_episode_created is called.
+        1) Single-/MultiAgentEpisode created: `on_episode_created()` is called.
         2) Respective sub-environment (gym.Env) is `reset()`.
-        3) Episode(V2)/Single-/MultiAgentEpisode starts: This callback is called.
+        3) Single-/MultiAgentEpisode starts: This callback is called.
         4) Stepping through sub-environment/episode commences.
 
         Args:
-            episode: The just started episode (after `env.reset()`). On the new API
-                stack, this will be a SingleAgentEpisode or MultiAgentEpisode object.
-                On the old API stack, this will be a Episode or EpisodeV2 object.
-            env_runner: Replaces `worker` arg. Reference to the current EnvRunner.
-            env: Replaces `base_env` arg.  The gym.Env (new API stack) or RLlib
-                BaseEnv (old API stack) running the episode. On the old stack, the
-                underlying sub environment objects can be retrieved by calling
-                `base_env.get_sub_environments()`.
-            rl_module: Replaces `policies` arg. Either the RLModule (new API stack) or a
-                dict mapping policy IDs to policy objects (old stack). In single agent
-                mode there will only be a single policy/RLModule under the
-                `rl_module["default_policy"]` key.
+            episode: The just started (after `env.reset()`) SingleAgentEpisode or
+                MultiAgentEpisode object.
+            env_runner: Reference to the EnvRunner running the env and episode.
+            metrics_logger: The MetricsLogger object inside the `env_runner`. Can be
```

Comment:
Awesome! I am looking forward to this. In my project I am using custom metrics and finally I can use them with EnvRunners